.github/workflows/drivers.yml

```diff
@@ -171,6 +171,42 @@ jobs:
       with:
         junit-name: 'be-tests-mysql-5-7-ee'
 
+  be-tests-mysql-latest-ee:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 35
```

Comment:
If this is taking 24 minutes for a normal run (which is totally crazy but nevermind that) we probably want a higher timeout than 35 minutes just to be safe. Last thing we want is it to be like 95% finished after 35 minutes and then get killed which would mean we'd have to run it a second time.

I've seen a lot of tests get killed for this reason actually so maybe it's a good idea to bump all of the driver tests to an hour timeout or so until we can figure out why they have gotten so slow recently

Reply:
This will do it :)
https://github.com/metabase/metabase/pull/27071/commits/6ef337280d049e43353061c37a8cc8f03504229e

Btw, @ariya suggested we start using beefier runners on ALL driver tests. We don't have to use buildjet anymore because GitHub has their own machines now. The cost is not going to be insignificant, though.